### DISCUSSION TITLE
Нужны идеи для развития pgadmin3

### DISCUSSION THREAD

**Discussion by levinsv** [levinsv/pgadmin3#29] [Ideas]
## 👋 Welcome!
 Если есть какое то полезное предложение что добавить, то пишите.

**Comment by pablotpy** [comment]
Estoy teniendo problemas, para nstalar, la guia no es muy especifica, podrias ayudarme?

**Comment by karasevdo** [comment]
Было бы неплохо запускать запрос через  F5, скрипт через F6. есть уже скомпилированная версия?

**Comment by levinsv** [comment]
Есть возможность задать свои комбинации клавиш для некоторых команд. [в этом коммите описание](https://github.com/levinsv/pgadmin3/commit/91a1160e13b43c8ba8daa8976cd7b751bedac044)
Скомпилированая версия тут [x64/Release_(3.0)]([x64/Release_(3.0)](https://github.com/levinsv/pgadmin3/tree/master/x64/Release_(3.0)))

**Reply by karasevdo** [reply]
Класс, спасибо! будем пробовать! 
Ссылка на папку с exe немного неверно отобразилась, но вот она [x64/Release_(3.0)](https://github.com/levinsv/pgadmin3/tree/master/x64/Release_(3.0))
